nltk/test/unit/translate/test_bleu.py

```diff
@@ -149,7 +153,9 @@ def test_case_where_n_is_bigger_than_hypothesis_length(self):
         # it's a special case where reference == hypothesis.
         references = ['John loves Mary'.split()]
         hypothesis = 'John loves Mary'.split()
-        assert(sentence_bleu(references, hypothesis, weights) == 1.0)
+        # Since no 4-grams matches were found the result should be zero
+        # exp(w_1 * 1 * w_2 * 1 * w_3 * 1 * w_4 * -inf) = 0
+        self.assertAlmostEqual(sentence_bleu(references, hypothesis, weights), 0.0, places=4)
```

Comment:
Please READ the hypothesis and reference properly and think about the values you're changing in the unittest. 

Unittest is meant to test the value or flow of the program. Change the values such that it suits the output your function is giving is just wrong =(